source/tutorials/nixos-configuration-on-vm.md

```diff
@@ -17,15 +17,17 @@ Virtual machines are a practical tool for debugging NixOS configurations.
 
 ## What do you need?
 
-- A working [Nix installation](install-nix) on Linux, or [NixOS](https://nixos.org/manual/nixos/stable/index.html#sec-installation)
+- A working [Nix installation](https://nixos.org/manual/nix/stable/installation/installation.html) on Linux, or [NixOS](https://nixos.org/manual/nixos/stable/index.html#sec-installation)
 - Basic knowledge of the [Nix language](reading-nix-language)
 
 ## Starting from the default NixOS configuration
 
 On NixOS, by default, the configuration file is located at `/etc/nixos/configuration.nix`.
-For commands that use a configuration file, such as `nixos-rebuild`, the configuration file's path can be specified.
+For commands that use a configuration file, such as `nixos-rebuild`, the configuration file's path can be specified.[^path]
```

Comment:
we need the footnote because we do not have a documentation to which we can link precisely. Without further information `can be specified` is derailing by itself.

Reply:
Maybe we can instead restructure the approach altogether. I like the idea how you start with the default configuration file and work your way to building the VM. What about something along these lines:

> On NixOS, by default, the configuration file is located at `/etc/nixos/configuration.nix`.
> In this tutorial we do not to install or update the current NixOS, but want to run a specific configuration in a virtual machine.
> 
> On NixOS, you can use the `nixos-generate-config` command to create a configuration file that contains some useful defaults and configuration suggestions.
> Create a NixOS configuration in your working directory:

BTW does `nixos-generate-config` even exist outside of NixOS these days? If not, you must provide an alternative. Or maybe better, take the case that always works (i.e. on any Linux) by default (e.g. present a finished minimal configuration) and make a footnote to mention `nixos-generate-config`. Although, thinking about it, even that doesn't really relate to what we're doing here, so we may as well leave it out. You know, if you start talking about `nixos-generate-config`, you suddenly have to perform additional steps just to get to a minimal configuration instead of just copy-pasting your sample.

Let's rather use that reader time to help them understand the sample.

Reply:
Really sorry for the long editing route, but I'd rather have multiple very short, on-point tutorials or guide solving one specific problem instead of covering all kinds of nice-to-know things that readers will be distracted by or won't care about.

In fact, there should be no nice-to-know things in the first place. All of the use cases should be explicit and discoverable from introductory material. We'll hopefully work on this at some point.

Reply:
We discussed the mismatch between tutorial/guide work and reference work [yesterday on the docs team](https://github.com/NixOS/nix.dev/issues/296#issuecomment-1271236970). Yeah, it's painful to get something done in terms of covering user stories if reference documentation is severely lacking, and at the very least it produces constant distraction. Still we should make some progress at all, to make Nix useful without users having to stick their head into random details.

Reply:
It's a footnote ready to be removed when references are ready